backend/frontend/templates/frontend/auth/page_permission_denied.html

```diff
@@ -0,0 +1,22 @@
+{% extends "frontend/user/base.html" %}
+{% block content %}
+  <div class="mx-auto w-1/3 p-6 bg-white rounded">
+    <h1 class="{{ styles.heading1 }}">Permission Denied</h1>
+    <hr class="my-2" />
+    
+    {% if messages %}
+      {% for message in messages %}
+        <div class="{{ message.extra_tags }}" role="alert">
+          <div class="text-sm leading-relaxed">{{ message.message }}</div>
+        </div>
+      {% endfor %}
+      <br />
+    {% endif %}
```

Comment:
I don't see what kind of messages you would need for this page other than the paragraph below.

I could be wrong :shrug: